classes/hunter.lua

```diff
@@ -29,6 +30,10 @@ local function registerClass(self)
     -- Counterattack, registered as both aura and counter, but only used as counter
     self:RegisterAura("counterattack", 0, counterattack, nil, "", 0, 0, 0, 0, false, { (GetSpellInfo(counterattack)) });
     self:RegisterCounter("counterattack"); -- Must match name from above call
+
+    -- Kill Shot, execute based on 20% and below
+    self:RegisterAura("killShot", 0, killShot, nil, "", 0, 0, 0, 0, false, { {GetSpellInfo(killShot)} });

```

Comment:
The glowIDs param should be `{ (GetSpellInfo(killShot)) }` instead of `{ {GetSpellInfo(killShot)} }` (parentheses vs. curly brackets) to be consistent with other calls of RegisterAura, such as counterattack a few lines above.

Reply:
Fixed